demo/react/src/VoiceWidget.tsx

```diff
@@ -0,0 +1,215 @@
+import React, { useCallback, useRef, useState } from "react";
+
+import { useLeopard } from "@picovoice/leopard-react";
+import { WebVoiceProcessor } from "@picovoice/web-voice-processor";
+import { PvEngine } from "@picovoice/web-voice-processor/dist/types/types";
```

Comment:
has to be imported like this?

Reply:
yes, web-voice-processor doesn't export `PvEngine`. Could just remove this and let react do type inference